source/source-connector/usage-examples/topic-naming.txt

```diff
@@ -117,10 +127,117 @@ mapping, the connector performs the following actions:
+   characters that need escaping. For example, to match
+   ``'.'``, the regex syntax requires that you escape it as ``'\.'``
+   You must escape the reverse solidus ``'\'`` character as ``'\\'``.
+   Consequently, to match ``'.'`` in your namespace pattern, you must
+   write it as ``'\\.'``.
+ 
+The connector matches namespaces by using the following order:
+ 
+1. Pairs with database and collection names in the namespace pattern. To
+   learn more about this namespace pattern, see the :ref:`Topic
+   Namespace Map Example <topic-naming-namespace-map-example>`.
+#. Pairs with only a database name in the namespace pattern. To
+   learn more about this namespace pattern, see the :ref:`Topic
+   Namespace Map Example <topic-naming-namespace-map-example>`.
+#. Regex pairs in order.
+#. The wildcard pair. To learn more about this namespace pattern, see
+   the :ref:`Topic Namespace Map with Wildcard Example
+   <topic-naming-namespace-map-wildcard-example>`.
+
+This example shows how to specify the ``topic.namespace.map``
+setting to define the following topic namespace mapping:
+
+- Maps a regular expression to the topic ``industrial{sep_coll}``. The
+  regex pattern matches any database names that include the string
+  ``'vertical'``, even if the database name has other characters.
+
+- Maps the ``vertical.health`` namespace to the ``healthcare`` topic.
+
+.. code-block:: ini
+
+   topic.namespace.map={"/vertical(?:\\..*)?": "industrial{sep_coll}", "vertical.health": "healthcare"}
+
+Since the ``vertical.health`` namespace mapping takes precedence over
+the regular expression namespace mapping, the connector performs the
+following actions:
+
+- If the change event comes from the database ``vertical`` and
+  collection ``health``, the connector sets the destination to the
+  ``healthcare`` topic.
+
+- If the change event comes from any database that gets
+  matched by the regex, the connector sets the destination to the
+  ``industrial.<collectionName>`` topic. The following examples
+  demonstrate this mapping:
+
+  - If the change event comes from the ``vertical.wasteManagement``
+    namespace, the connector writes to the
+    ``industrial.wasteManagement`` topic.
+
+  - If the change event comes from the ``vertical_auto.batteries``
+    namespace, the connector writes to the
+    ``industrial.batteries`` topic because the database name contains
+    the string ``'vertical'``.
+  
+  - If the change event comes from the ``vertical_auto``
+    database but no specific collection, the connector writes to the
+    ``industrial`` topic.
+
+- If the change document comes from any database that does not get
+  matched by the regex, the connector sets the destination topic to the
+  default namespace naming scheme. 
+
+- If defined, the connector applies the ``topic.prefix`` and
+  ``topic.suffix`` settings to the destination topic name after it
+  performs namespace mapping.
```

Comment:
I: I think this means if these settings are defined, but as written, it means "If the connector is defined..."

```suggestion
- If the ``topic.prefix`` and ``topic.suffix`` settings are defined, the connector applies
  their values  to the destination topic name after it
  performs namespace mapping.
```